### PR TITLE
[RFC] ci: work around "pyenv: pip3: command not found"

### PR DESCRIPTION
**Pull request by mhinz**

### CODE REVIEW

File: ci/install.sh [hunk 1]
```diff
@@ -14,12 +14,10 @@ fi
 
 # Use default CC to avoid compilation problems when installing Python modules.
 echo "Install neovim module and coveralls for Python 2."
-CC=cc pip2.7 -q install --user --upgrade neovim cpp-coveralls
+CC=cc pip2 -q install --user --upgrade neovim cpp-coveralls
 
 echo "Install neovim module for Python 3."
-# Allow failure. pyenv pip3 on travis is broken:
-# https://github.com/travis-ci/travis-ci/issues/8363
```
Comment (justinmk) [inline ci/install.sh]: That issue is still open. What changed?

Comment (mhinz) [reply ci/install.sh]: I'm still playing around with a few solutions. But I'll add the URL to that issue above the `pyenv globals` line.

Comment (justinmk) [reply ci/install.sh]: Fwiw It does work sometimes, check multiple builds. And the git log history for these lines.

**Comment by mhinz** [comment]
`pyenv global` won't help. Closing.

Ref: https://github.com/neovim/neovim/pull/7253

**Pull request closed without merging**